Emma/emma.py

```diff
@@ -51,7 +51,7 @@ def main(arguments):
     if memoryManager.settings.createCategories:
         sc().info("No results were generated since categorisation option is active.")
     else:
-        memoryManager.createReports(arguments.teamscale)
+        memoryManager.createReports(arguments.teamscale, arguments.noreports)
```

Comment:
```suggestion
        memoryManager.createReports(arguments.teamscale, arguments.dryRun)
```

Also change for all other appearances.